JSDemos/Demos/Gauges/Overview/Vue/description.md

```diff
@@ -1 +1,3 @@
-DevExtreme Vue Gauge controls are animated UI components designed for use in&nbsp;dashboards. The gauges can be&nbsp;of&nbsp;two geometric shapes: circular, or&nbsp;radial, and linear. You can use Vue syntax and techniques to&nbsp;instantiate and configure the gauges or&nbsp;handle their events. In&nbsp;addition, the components support [prop validation](https://vuejs.org/v2/guide/components-props.html#Prop-Validation). [Find out more about DevExtreme Vue components](/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/).
\ No newline at end of file
+DevExtreme Vue Gauge controls are animated UI components designed for use in dashboards. The gauges can be of two geometric shapes: circular, or radial, and linear. You can use Vue syntax and techniques to instantiate and configure the gauges or handle their events. In addition, the components support [prop validation](https://vuejs.org/v2/guide/components-props.html#Prop-Validation). [Find out more about DevExtreme Vue components](/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/).
+
+Check out other demos in this section to explore gauge features, like geometry customization, palettes, tooltips, custom labels in the component's center, etc.
```

Comment:
I explain the changes in the jQuery description (making descriptions consistent): 

```suggestion
DevExtreme Vue Gauge controls help you visualize data and create dashboards. Our library includes three gauge types: radial bar gauges, circular scale gauges, and linear gauges. 

This demo displays a simple dashboard with a few highly customized gauges. 

Review other demos in this section to explore gauge features such as geometry customization, palettes, tooltips, and custom label display.

You can use Vue syntax and techniques to instantiate and configure the gauges or handle their events. In addition, the components support [prop validation](https://vuejs.org/v2/guide/components-props.html#Prop-Validation). [Find out more about DevExtreme Vue components](/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/).
```